docs/_config.yml

```diff
@@ -4,4 +4,13 @@ defaults:
       path: "" # an empty string here means all files in the project
     values:
       layout: "default"
+      title: Expensify Help
+      tagline: Find all the answers to all your Expensify questions, as well as support for user and admin questions from the Expensify Community
+      description: The Community is a great place for Expensify users and experts to help one another, access training and implementation advice, discuss best practice approaches and contribute their feature and functionality ideas.
```

Comment:
cc @maddylewis @johncschuster can you guys help with an updated description? I suck at writing copy 😞